### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by FahadIqbal12**
## Fixes Issue
Added options for Layer 2 Solutions.

## Screenshots

Screenshot 2023-03-21 224002

### CODE REVIEW

File: Blockchain/readme.md [hunk 1]
```diff
@@ -44,6 +44,11 @@
   - [2)Infura](https://infura.io/)
   - [3)Quicknode](https://www.quicknode.com/)
 
+## Layer 2 Solutions
+- [Polygon](https://wiki.polygon.technology/)
+- [Lightning Network](http://lightning.network/docs/)
+
```
Comment (TRohit20) [inline Blockchain/readme.md:50]: Remove the empty spaces here, They seem to be unnecessary and redundant

Comment (FahadIqbal12) [reply Blockchain/readme.md:50]: Done

**Review by TRohit20 — APPROVED** [review]

File: Blockchain/readme.md [hunk 2]
```diff
@@ -109,3 +113,4 @@
   - [Austin Griffith](https://www.youtube.com/channel/UC_HI2i2peo1A-STdG22GFsA)
   - [Nader Dabit](https://www.youtube.com/user/boyindasouth)
   - [Harkirat Singh (Youtube)](https://www.youtube.com/playlist?list=PLVKLWop9wWA82pZoyylZD2VF2c7MR8_5I)
+  - [Code Eater (Hindi)(Youtube)](https://www.youtube.com/@CodeEater21)
```
Comment (siddhant-khisty) [inline Blockchain/readme.md:116]: Please remove this resource. We are trying to cater to an international audience, hence we want to keep all resources in English ONLY.

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

File: Blockchain/readme.md (hunk 2 above)
Comment (FahadIqbal12) [reply Blockchain/readme.md]: Done

**Review by siddhant-khisty — APPROVED** [review]

**Pull request merged**